docs/README.md

```diff
@@ -198,19 +198,17 @@ Next:
 Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
 
 <!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
-<!-- prettier-ignore-start -->
```

Comment:
@belozer that is ok?

Reply:
Hmm.... This `@all-contribute` did automatically.